versions/draft-3/SPEC.md

```diff
@@ -705,6 +721,11 @@ task t {
   - Two input files which originated in the same storage directory must also be localized into the same directory for task execution (see the special case handling for Versioning Filesystems below).
 - When a WDL author uses a `File` input in their [Command Section](#command-section), the fully qualified, localized path to the file is substituted into the command string.
 
+##### FileURI localization
+`FileURI` inputs also follow specific localization rules:
+- If the URI in the FileURI is a file mounted on the filesystem (e.g. `file:///home/foo/test.txt` or `/home/foo/test.txt`), then the File referenced at must be made accessible to the job at execution time by the engine.
```

Comment:
typo, should be `"...the File referenced must..."`

Reply:
Done